lib/spark.js

```diff
@@ -202,6 +217,29 @@ export default class Spark {
     }
   }
 
+  async testNetworkRetrieval(providers, cid, stats) {
+    if (!providers.length) {
+      console.warn('No providers found for the CID. Skipping network-wide retrieval check.')
```

Comment:
This shouldn't be a warning - it's an expected situation that some CIDs may not have any alternate retrieval providers supporting Graphsync or HTTP.

Let's reword the message to not say "network-wide".

```suggestion
      console.info('No alternative providers found for the CID.')
```